tools/migration/cmd/go-filecoin-migrate.go

```diff
@@ -0,0 +1,105 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/filecoin-project/go-filecoin/tools/migration/internal"
+	"github.com/filecoin-project/go-filecoin/tools/migration/migrate_1-to-2"
+)
+
+type Migrator interface {
+	Migrate(oldRepo, newRepo *os.File) error
+	Describe()
+	Validate(oldRepo, newRepo *os.File) error
+}
+
+// Run runs the given migration command
+func Run(cmd string, verbose bool) error {
+	var err error
+	miglog, err := makeMigl()
+	logpath := "/tmp/migration_log.txt"
```

Comment:
Blocking: turn this run method into a `MigrationRunner` struct and extract all these paths as member fields, configurable at construction.

It is confusing that as a file in the `cmd` package this looks like a command, but is in fact a router to other commands. I like the idea of identifying the commands clearly, but in this case they probably map to simple method calls on `Migrator`. This file might be better named `migration/migrator.go`.

Reply:
@anorth  in #2614 all this code has been ripped out and the file moved, but there is a stubbed-out MigrationRunner now for filling out in a follow-up PR.